test/unit/ngsiService/active-devices-test.js

```diff
@@ -391,6 +391,99 @@ describe('Active attributes test', function() {
         });
     });
 
+    describe('When the IoTA gets a set of values with a TimeInstant which are in ISO8601 format ' +
+        'without milis', function() {
+        var modifiedValues;
+
+        beforeEach(function(done) {
+            var time = new Date(1666477342000); // 2022-10-22T22:22:22Z
+
+            modifiedValues = [
+                {
+                    name: 'state',
+                    type: 'Boolean',
+                    value: 'true'
+                },
+                {
+                    name: 'TimeInstant',
+                    type: 'ISO8601',
+                    value: '2022-10-22T22:22:22Z'
```

Comment:
You may argue that this we already have a test for this, but the difference is that this one doesn't use miliseconds while the exsiting one use it. Thus, they are two different variants and I think it worth to keep both them.

(The same for the NGSIv2 tests)

Reply:
I agree! NTC

Reply:
Fixed in 9838184 (test removed)